rust/kcl-lib/src/std/assert.rs

```diff
@@ -24,132 +24,124 @@ async fn _assert(value: bool, message: &str, args: &Args) -> Result<(), KclError
     } else {
-        Err(KclError::Type(KclErrorDetails {
-            message: format!("assert failed because {left} != {right}: {message}"),
-            source_ranges: vec![args.source_range],
-        }))
+        Default::default()
+    };
+    let actual = actual.n;
+    if let Some(exp) = is_greater_than {
+        let exp = exp.n;
+        _assert(
+            actual > exp,
+            &format!("Expected {actual} to be greater than {exp} but it wasn't{suffix}"),
+            args,
+        )
+        .await?;
     }
-}
-
-pub async fn assert_equal(_exec_state: &mut ExecState, args: Args) -> Result<KclValue, KclError> {
-    let (left, right, epsilon, description): (TyF64, TyF64, TyF64, String) = args.get_data()?;
-    inner_assert_equal(left.n, right.n, epsilon.n, &description, &args).await?;
-    Ok(KclValue::none())
-}
-
-/// Check that a numerical value is greater than another at runtime,
-/// otherwise raise an error.
-///
-/// ```no_run
-/// assertGreaterThan(2, 1, "2 is greater than 1")
-/// ```
-#[stdlib {
-    name = "assertGreaterThan",
-}]
-async fn inner_assert_gt(left: f64, right: f64, message: &str, args: &Args) -> Result<(), KclError> {
-    _assert(left > right, message, args).await
-}
-
-pub async fn assert_lte(_exec_state: &mut ExecState, args: Args) -> Result<KclValue, KclError> {
-    let (left, right, description): (TyF64, TyF64, String) = args.get_data()?;
-    inner_assert_lte(left.n, right.n, &description, &args).await?;
-    Ok(KclValue::none())
-}
-
-/// Check that a numerical value is less than or equal to another at runtime,
-/// otherwise raise an error.
-///
-/// ```no_run
-/// assertLessThanOrEq(1, 2, "1 is less than 2")
-/// assertLessThanOrEq(1, 1, "1 is equal to 1")
-/// ```
-#[stdlib {
-    name = "assertLessThanOrEq",
-}]
-async fn inner_assert_lte(left: f64, right: f64, message: &str, args: &Args) -> Result<(), KclError> {
-    _assert(left <= right, message, args).await
-}
-
-pub async fn assert_gte(_exec_state: &mut ExecState, args: Args) -> Result<KclValue, KclError> {
-    let (left, right, description): (TyF64, TyF64, String) = args.get_data()?;
-    inner_assert_gte(left.n, right.n, &description, &args).await?;
-    Ok(KclValue::none())
-}
-
-/// Check that a numerical value is greater than or equal to another at runtime,
-/// otherwise raise an error.
-///
-/// ```no_run
-/// assertGreaterThanOrEq(2, 1, "2 is greater than 1")
-/// assertGreaterThanOrEq(1, 1, "1 is equal to 1")
-/// ```
-#[stdlib {
-    name = "assertGreaterThanOrEq",
-}]
-async fn inner_assert_gte(left: f64, right: f64, message: &str, args: &Args) -> Result<(), KclError> {
-    _assert(left >= right, message, args).await
+    if let Some(exp) = is_less_than {
+        let exp = exp.n;
+        _assert(
+            actual < exp,
+            &format!("Expected {actual} to be less than {exp} but it wasn't{suffix}"),
+            args,
+        )
+        .await?;
+    }
+    if let Some(exp) = is_greater_than_or_equal {
```

Comment:
Seems like you can specify `isGreaterThanOrEqual` and `epsilon`, but in this case, `epsilon` is ignored. Maybe it should be an error to specify `epsilon` when it isn't used.